package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
@timbur the error I get on master is `Uncaught (in promise) Error: imports[1] for react-transform-catch-errors does not look like a React component.` is that the one you mean?